pkgs/development/python-modules/cramjam/default.nix

```
@@ -13,29 +16,52 @@
 
 buildPythonPackage rec {
   pname = "cramjam";
-  version = "2.8.3";
+  version = "2.9.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "milesgranger";
-    repo = "pyrus-cramjam";
+    repo = "cramjam";
     tag = "v${version}";
-    hash = "sha256-1KD5/oZjfdXav1ZByQoyyiDSzbmY4VJsSJg/FtUFdDE=";
+    hash = "sha256-s+dKoZftI+aXcie1quAo/Xmlt1BQrzoCjwesDiaNABY=";
   };
 
+  postPatch =
+    # Unclear why dynamic library loading is not the default.
+    #
+    # error: could not find native static library `isal`, perhaps an -L flag is missing?
+    ''
+      substituteInPlace "$cargoDepsCopy"/isal-sys-0.5.3+496255c/build.rs \
```

```suggestion
      substituteInPlace "$cargoDepsCopy"/isal-sys-*/build.rs \
```

As the substitution isn't version-specific